### PR TITLE
ACM-12797 - LaunchLogs uses default path for installer pod logs

### PR DESCRIPTION
**Pull request by Randy424**
Regarding: [ACM-12899](https://issues.redhat.com/browse/ACM-12899)

https://github.com/user-attachments/assets/ed099cca-aaee-4221-bb4a-de02d57d0c3d

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx [hunk 1]
```diff
@@ -107,11 +107,8 @@ export function launchLogs(cluster: Cluster, configMaps: ConfigMap[]) {
     const response = getHivePod(cluster.namespace, cluster.name, cluster.status)
     response.then((job) => {
       const podName = job?.metadata.name || ''
-      const containerName = podName.includes('uninstall') ? 'deprovision' : 'hive'
-      podName &&
-        window.open(
-          `${openShiftConsoleUrl}/k8s/ns/${cluster.namespace}/pods/${podName}/logs?container=${containerName}`
-        )
+      const containerName = podName.includes('uninstall') ? '?container=deprovision' : ''
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx:110]: Can you check if we can remove the container name for deprovision as well?

Comment (Randy424) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx]: It looks like the default for the uninstall pod also works.

**Pull request merged**